packages/nimble-components/src/theme-provider/tests/tokens.stories.ts

```diff
@@ -104,12 +104,17 @@ const tokenTemplates: {
     FontSize: stringValueTemplate,
     TextTransform: stringValueTemplate,
     FontFamily: stringValueTemplate,
+    BoxShadow: stringValueTemplate,
+    OffsetX: stringValueTemplate,
+    OffsetY: stringValueTemplate,
```

Comment:
The intention of these is to represent the different units. The "units" of OffsetX and OffsetY. If we had a redo chance I'd probably merge Size, Width, Height, and maybe FontSize into just "Length" or just "Size" since they are all css lengths:

https://developer.mozilla.org/en-US/docs/Web/CSS/length

so to prevent a new unit I think these should be a Size prefix, i.e. `elevation3BoxShadowOffsetYSize`